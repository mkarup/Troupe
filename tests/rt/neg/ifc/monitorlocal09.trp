let val p = spawn (fn () => sleep 750)
    val monRef = monitorlocal (p, `{secret}`)
in
  adv ()
end
  