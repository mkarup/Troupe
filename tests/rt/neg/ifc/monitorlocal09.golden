2024-12-11T16:27:06.494Z [RTM] [32minfo[39m: Skipping network creation. Observe that all external IO operations will yield a runtime error.
Runtime error in thread 148cffaa-9295-4164-ab7b-68ebb6721b55@{}%{}
>> Illegal flow in adv function:
 |    pc: {}
 | block: {secret}
 | value: ()@{}%{}
